ext/intl/timezone/timezone_methods.cpp

```diff
@@ -180,15 +180,15 @@ U_CFUNC PHP_FUNCTION(intltz_create_enumeration)
 		se = TimeZone::createEnumeration(Z_STRVAL_P(arg));
 	} else {
 		intl_error_set(NULL, U_ILLEGAL_ARGUMENT_ERROR,
-			"intltz_create_enumeration: invalid argument type", 0);
+			"invalid argument type", 0);
 		RETURN_FALSE;
 	}
```

Comment:
This feels like it should be a TypeError, something to remember as future improvements

Reply:
Actually this looks like it should just use Fast ZPP with a int|string type check.